packages/app-headless-cms/src/admin/components/ContentEntryForm/Header/DeleteEntry/useDeleteEntry.tsx

```diff
@@ -24,12 +24,12 @@ export const useDeleteEntry = (): UseDeleteEntryDialogResponse => {
 
     const title = get(entry, "meta.title");
     const { showConfirmation } = useConfirmationDialog({
-        title: "Delete content entry",
+        title: "Trash entry",
         message: (
             <p>
-                You are about to delete this content entry and all of its revisions!
+                Are you sure you want to trash <strong>{title}</strong>?
```

Comment:
> Are you sure you want to trash

this sounds awkward. Maybe `Are you sure you want to move XYX to trash?` I'm not sure you can use `to trash` as a verb like that.